scripts/quality/benchmark_cairo_auditor.py

```
@@ -12,6 +13,56 @@
+        / "quality"
+        / "component_resolution.py"
+    )
+    if not module_path.is_file():
+        return None
+    spec = importlib.util.spec_from_file_location("cairo_auditor_component_resolution", module_path)
+    if spec is None or spec.loader is None:
+        return None
+    module = importlib.util.module_from_spec(spec)
+    try:
+        spec.loader.exec_module(module)
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)